### PR TITLE
Fixed preview events for media-selection

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ CHANGELOG for Sulu
 
 * dev-master
     * HOTFIX      #1511 [ContentBundle]  Fixed single-internal-link overlay URL
+    * HOTFIX      #1521 [MediaBundle]     Fixed media-selection events for preview update
```
Comment (danrot) [inline CHANGELOG.md]: Line not aligned

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/validation/types/mediaSelection.js [hunk 2]
```diff
@@ -15,8 +15,10 @@ define([
     'use strict';
 
     var dataChangedHandler = function(data, $el) {
-        App.emit('sulu.preview.update', $el, data);
-        App.emit('sulu.content.changed');
+        if (!!data && !!$el) {
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/validation/types/mediaSelection.js]: Aren't you  just fixing symptoms here? To me the real issue looks like the `sulu.media-selection.images.data-changed` event is thrown twice, once even missing the data.

Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/validation/types/mediaSelection.js]: It looks to me like the real issue is here: https://github.com/sulu-io/sulu/blob/develop/src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/main.js#L246

File: CHANGELOG.md (hunk 1 above)
Comment (wachterjohannes) [inline CHANGELOG.md]: this is in the same line (:

Comment (wachterjohannes) [inline CHANGELOG.md]: ok i see it (:

**Pull request merged**